libs/core/src/lib/button-group/button-group.component.scss

```diff
@@ -1,3 +1,3 @@
 // @import '~fundamental-styles/dist/button-group';
 @import '~fundamental-styles/dist/button';
-@import '~fundamental-styles/dist/icon';
+@import "~fundamental-styles/dist/icon";
```

Comment:
kinda strange that it changed to double quote 
![image](https://user-images.githubusercontent.com/10849982/80716740-4efdcd80-8af8-11ea-9986-edfbc2ea4c01.png)
(btw button-group styles are commented out but I'll handle that)